java-frontend/src/test/java/org/sonar/java/model/JavaVersionImplTest.java

```diff
@@ -85,9 +85,9 @@ void compatibilityMesssages() {
 
   @Test
   void test_effective_java_version() {
-    assertThat(new JavaVersionImpl().effectiveJavaVersionAsString()).isEqualTo("18");
+    assertThat(new JavaVersionImpl().effectiveJavaVersionAsString()).isEqualTo("19");
```

Comment:
There is a parametrized tests a few lines above, which test all the versions we support. What about adding `19` as well?